tests/unit/utils/test_ssdp.py

```diff
@@ -14,6 +14,7 @@
 from salt.utils import ssdp
 import datetime
 from salt.ext.six.moves import zip
+from salt.ext.six import text_type as str
```

Comment:
Since we will eventually be linting for `str()`, we'd prefer using six.text_type in the code instead of replacing the builtin like that.

Reply:
@terminalmage OK, fixd.

Reply:
We should be importing `six` and using `six.text_type` instead of importing `text_type` directly into the module's global namespace. It's not as important here in the tests, but it's very important in the salt code itself. The reason for this is because our loader will add any functions in the global namespace to the LazyDict. So, if this were in salt/utitls/ssdp.py we'd have a function called `__utils__['ssdp.text_type']()`.

This wasn't important enough to delay merging, though. Since the tests passed and everything else was fine, I took care of changing this import in 7115cd9d as part of a manual merge.